packages/ui-components/stencil.config.ts

```diff
@@ -3,14 +3,14 @@ import { Config } from '@stencil/core';
 import { reactOutputTarget as react } from '@stencil/react-output-target';
 import { sass } from '@stencil/sass';
 
+// Configuration to generate properties with 2 way bind in angular
 const angularValueAccessorBindings: ValueAccessorConfig[] = [
-	// Configuration example to generate properties with 2 way bind in angular
-	// {
-	// 	elementSelectors: ['component-name'],
-	// 	event: 'eventNameOfChamge',
-	// 	targetAttr: 'propName',
-	// 	type: 'number/type'
-	// }
+	{
+		elementSelectors: ['kv-switch-button'],
+		event: 'switchStateChange',
+		targetAttr: 'state',
+		type: 'text'
```

Comment:
@miguelpinto98 

Since this is an enum it could make sense to change but sincerely I don't know the implications of changing it. Do you know?

Reply:
I think is related with HTML elements types, like radio, select, checkbox, number, text... And convert it to it. We can go with this, and text a different approach if we need.

Reply:
Ok gotcha, thanks 👍